tests/testthat/test-pstr_prepare_companies.R

```diff
@@ -1,3 +1,4 @@
+# TODO: Paarameter `col_types = cols(isic_4digit = col_character())` is missing
 test_that("snapshot", {
   withr::local_options(list(readr.show_col_types = FALSE))
```

Comment:
@maurolepore I guess this PSTR snapshot test does not include the parameter `col_types = cols(isic_4digit = col_character())`. Please correct me if I am wrong